Moli/Core/Screens/Home/Live/MoliRoomItemView.swift

```diff
@@ -0,0 +1,111 @@
+import SwiftUI
+
+struct MoliRoomItemView: View {
+    var room: API.Room.HomeRoomResponse
+    
+    @State private var hosts: [API.Room.HostInfo] = []
+    @State private var viewerProfiles: [API.User.UserMinified] = []
+    @State private var extraViewersCount = 0
+    
+    private func setHostProfiles() {
+        self.hosts = room.hostsInfo
+    }
+    
+    private func setViewerProfiles() {
+        let viewerProfiles = Array(room.previewViewers.prefix(4))
+        
+        self.viewerProfiles = API.User.UserMinified.convertFrom(viewerProfiles)
+        self.extraViewersCount = self.room.viewers.count - viewerProfiles.count
+    }
+    
+    var body: some View {
+        ZStack(alignment: .topLeading) {
+            if let imageURL = room.thumbnail?.thumbnailUrl, let url = URL(string: imageURL) {
+                AsyncImage(url: url) { image in
+                    image.resizable()
+                        .ignoresSafeArea()
+                } placeholder: {
+                    Image("moliRoomsFallbackBG")
+                        .resizable()
+                        .ignoresSafeArea()
+                }
+            }
+            VStack(alignment: .leading) {
+                HStack {
+                    Text(room.title.replacingOccurrences(of: " ", with: "\n").uppercased())
+                        .font(Font.headingLarge.weight(.bold))
+                        .multilineTextAlignment(.leading)
+                        .padding(8)
+                        .background(room.title == "GRWM" ? Color.newYellow2 : Color.newPurple)
+                        .cornerRadius(16)
+                        .foregroundColor(.black)
+                    Spacer()
+                }
```

Comment:
:art: I det här fallet ska du inte behöva någon HStack för att placera texten med rätt layout. Dock såg jag att när man tar bort den här så centreras allt till mitten, men det beror på modifier:n som ligger längre ner `.frame(maxWidth: .infinity)` som behöver ändras till `.frame(maxWidth: .infinity, alignment: .leading)` så att modifier:n vet att du vill ha allt content på .leading delen av kortet

```suggestion
                    Text(room.title.replacingOccurrences(of: " ", with: "\n").uppercased())
                        .font(Font.headingLarge.weight(.bold))
                        .multilineTextAlignment(.leading)
                        .padding(8)
                        .background(room.title == "GRWM" ? Color.newYellow2 : Color.newPurple)
                        .cornerRadius(16)
                        .foregroundColor(.black)
```